crates/router/src/core/customers.rs

```diff
@@ -141,7 +141,7 @@ pub async fn delete_customer(
         Ok(customer_payment_methods) => {
             for pm in customer_payment_methods.into_iter() {
                 if pm.payment_method == enums::PaymentMethod::Card {
-                    cards::delete_card_from_hs_locker(
+                    cards::delete_card_wrapper(
```

Comment:
change the function name, this is not a wrapper function, `delete_card_from_locker` locker can be `legacy` or `basilisk`?